opm/autodiff/MissingFeatures.hpp

```diff
@@ -24,7 +24,7 @@ namespace Opm {
 
 namespace MissingFeatures {
 
-    void checkKeywords(std::shared_ptr<const Deck> deck, std::shared_ptr<const Parser> parser);
```

Comment:
It's not a very big deal at this point, since we have not done the "big sweep of shared pointers from parser" transformation yet, but @jokva is right: the method only requires a const reference.